comictaggerlib/ui/seriesselectionwindow.ui

```diff
@@ -85,17 +146,11 @@
            <property name="text">
             <string>Year</string>
            </property>
-           <property name="textAlignment">
-            <set>AlignCenter</set>
-           </property>
           </column>
           <column>
            <property name="text">
             <string>Issues</string>
            </property>
-           <property name="textAlignment">
-            <set>AlignCenter</set>
-           </property>
```

Comment:
Is there a reason text alignment is changing here?

Reply:
I believe this is because `center` is the default and so Qt Designer removed it when I saved as it's superfluous. If I open it from `develop` and save, it does the same thing.

Reply:
I must have an old version of QT Designer...